modules/map/tools/map_datachecker/client/main.cc

```diff
@@ -0,0 +1,42 @@
+/******************************************************************************
+ * Copyright 2019 The Apollo Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *****************************************************************************/
+#include "cyber/cyber.h"
+#include "modules/map/tools/map_datachecker/client/client.h"
+#include "modules/map/tools/map_datachecker/client/client_gflags.h"
+
+int main(int argc, char** argv) {
+  fprintf(stderr, "parsing command lines\n");
+  google::ParseCommandLineFlags(&argc, &argv, true);
+  fprintf(stderr, "parsing command lines done\n");
```

Comment:
1. Seems like a information message instead of error.
1. Use `AINFO`, `AERROR`, or at least std::cout, std::cerr, instead of C style printf.